classes/class-wc-connect-shipping-label.php

```diff
@@ -107,12 +107,12 @@ protected function get_form_data( WC_Order $order ) {
 			$form_data[ 'destination' ] = array(
 				'name' => trim( $dest_address[ 'first_name' ] . ' ' . $dest_address[ 'last_name' ] ),
 				'company' => $dest_address[ 'company' ],
-				'address_1' => $dest_address[ 'address_1' ],
+				'address' => $dest_address[ 'address_1' ],
```

Comment:
Or maybe the server should also be changed to match WooCommerce?
Anyways, we can put this off for later.